robot_skills/src/robot_skills/util/kdl_conversions.py

```diff
@@ -87,6 +87,9 @@ def __eq__(self, other):
         else:
             return False
 
+    def __ne__(self, other):
+        return not self.__eq__(other)
```

Comment:
CI wasn't working. I found out __ne__ is not defined for the vector stamped class. Could place this in another PR but its a small fix